content/concepts/introduction/protocols/ping.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Ping"
+description: "The ping protocol is a simple request response protocol."
+weight: 20
+---
+
+## What is Ping?
+
+Ping is a network utility used to test a node's reachability.
```

Comment:
This is ICMP ping, not libp2p ping. You can only libp2p ping if you already have a libp2p connection. It’s worth noting here that these are completely different things though.